geomesa-process/src/main/scala/org/locationtech/geomesa/process/ProcessFactory.scala

```diff
@@ -36,5 +37,7 @@ class ProcessFactory
     classOf[SamplingProcess],
     classOf[JoinProcess],
     classOf[RouteSearchProcess],
-    classOf[BinConversionProcess]
+    classOf[BinConversionProcess],
+    classOf[ArrowConversionProcess]
```

Comment:
@aheyne As we fix up the process bits, we should make a clear suggestion about how to change this fille (if necessary).